ydb/core/blobstorage/dsproxy/dsproxy_strategy_base.cpp

```diff
@@ -363,10 +363,11 @@ void TStrategyBase::Evaluate3dcSituation(const TBlobState &state,
     }
 }
 
-void TStrategyBase::Prepare3dcPartPlacement(const TBlobState &state,
+bool TStrategyBase::Prepare3dcPartPlacement(const TBlobState &state,
```

Comment:
Return fullPlacement through out parameter. Otherwise there is a mixup of return values and return parameters -- it is not obvious what this function returns (based on its name).